File: Python/Basic Maths/swapNumbers.py
# program to swap two variables using a temporary variable

x = input('Enter value of x: ')
y = input('Enter value of y: ')
temp = x
x = y
y = temp
print(f'The value of x after swapping: {x}')
print(f'The value of y after swapping: {y}')


#============================================================================
#============================================================================


# program to swap two variables


x = input('Enter value of x: ')
y = input('Enter value of y: ')
x, y = y, x
print(f'The value of x after swapping: {x}')
print(f'The value of y after swapping: {y}')


#============================================================================
#============================================================================

File: C++/Basic Maths/swapNumbers.cpp
#include <iostream>
using namespace std;

// program to swap two variables

int main() {
    int x, y, temp;
    cout << "Enter two numbers: ";
    cin >> x >> y;
    cout << "Before swapping x = " << x << " and y = " << y << endl;
    temp = x;
    x = y;
    y = temp;
    cout << "After swapping x = " << x << " and y = " << y << endl;
    return 0;
}